src/components/TagEditor.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import {connect} from "react-redux";
+import Container from 'react-bootstrap/Container'
+import DragDropColumn from "./DragDropColumn";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import {DragDropContext} from "react-beautiful-dnd";
+
+class TagEditor extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
```

Comment:
Cool, so the next steps would be changing this state variable to retrieve tags and their content from the backend?